specs/http-api/README.md

```diff
@@ -166,14 +147,37 @@ Pagination is supported using the following query params:
 ---
 
 # Idempotency
-The IDs of individual tbDEX messages are used as idempotency keys
+The IDs of individual tbDEX messages are used as idempotency keys. For example, in a tbDEX Order message:
+
+```json
+{
+  "metadata": {
+    "from": "did:key:z6MkvUm6mZxpDsqvyPnpkeWS4fmXD2jJA3TDKq4fDkmR5BD7",
+    "to": "did:ex:pfi",
+    "exchangeId": "rfq_01ha83pkgnfxfv41gpa44ckkpz",
+    "kind": "order",
+    "id": "order_01ha83pkgsfk6t1kxg7q42s48j",
+    "createdAt": "2023-09-13T20:28:40.345Z",
+    "protocol": "1.0"
+  },
+  "data": {},
```

Comment:
is data present even if it's a null value?

Reply:
yep! order's data is empty.